tests/test_pfcwd.py

```diff
@@ -0,0 +1,98 @@
+import redis
+import time
+import os
+import pytest
+import re
+import json
+from swsscommon import swsscommon
+
+PFCWD_TABLE_NAME = "DROP_TEST_TABLE"
+PFCWD_TABLE_TYPE = "DROP"
+PFCWD_TC = ["3", "4"]
+PFCWD_RULE_NAME_1 =  "DROP_TEST_RULE_1"
+PFCWD_RULE_NAME_2 =  "DROP_TEST_RULE_2"
+
+class TestPfcWd:
+    def find_acl_rule_oid(self, dvs):
+       self.asic_db = swsscommon.DBConnector(1, dvs.redis_sock, 1)
+       atbl = swsscommon.Table(self.asic_db, "ASIC_STATE:SAI_OBJECT_TYPE_ACL_ENTRY")
+       keys = atbl.getKeys()
+       acl_entry = [k for k in keys if k not in dvs.asicdb.default_acl_entries]
+
+       return acl_entry[0]
+
+    def test_PfcWdAclCreationDeletion(self, dvs, dvs_acl, testlog):
+        try:
+            dvs_acl.create_acl_table(PFCWD_TABLE_NAME, PFCWD_TABLE_TYPE, ["Ethernet0","Ethernet8", "Ethernet16", "Ethernet24"], stage="ingress")
+            dvs.runcmd("counterpoll queue enable")
+            time.sleep(10)
```

Comment:
This is OK for this PR, but it would be better to poll for some event (like the QUEUE_NAME_MAP appearing in the DB) that indicates this command was successful rather than adding a static 10s sleep.

Reply:
I also don't know if this step is still necessary for this PR or not given that we're going through config DB directly instead of triggering the PFCWD.